client/extensions/woocommerce/index.js

```diff
@@ -14,31 +14,88 @@ import ProductCreate from './app/products/product-create';
 import Dashboard from './app/dashboard';
 import StatsController from './app/stats/controller';
 
-const Controller = {
-	dashboard: function( context ) {
+function addStorePage( storePage ) {
+	page( storePage.route, siteSelection, navigation, function( context ) {
```

Comment:
This breaks store stats. `StatsController` is a function which calls `renderWithReduxStore` after doing some route validation. 

Could the callback be `storePage.controller || function ( context) { ... }`? Where 
```js
{
  controller: StatsController,
  configKey: 'woocommerce/extension-stats',
  route: '/store/stats/:type/:period/:site',
},